<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWidget</class>
 <widget class="QDockWidget" name="DockWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>861</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DockWidget</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>0</y>
      <width>431</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>427</width>
          <height>707</height>
         </rect>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>431</width>
           <height>711</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_4" rowstretch="10,0,0,0,0,0,0,0,0">
          <property name="verticalSpacing">
           <number>7</number>
          </property>
          <item row="8" column="0" colspan="2">
           <widget class="QTableWidget" name="table_recap">
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Unknowns</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Exploitation (kgCO2eq/an)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Construction (kgCO2eq)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Detail level</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="model_label">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QWidget" name="pie_chart_e" native="true"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="recap_label">
            <property name="text">
             <string>RÃ©capitulatif</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QWidget" name="bar_chart_e" native="true"/>
          </item>
          <item row="1" column="1">
           <widget class="QWidget" name="pie_chart_c" native="true"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QWidget" name="bar_chart_c" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
